apps/studio/components/interfaces/Support/SupportForm.tsx

```diff
@@ -40,9 +40,10 @@ const INCLUDE_DISCUSSIONS = ['Problem', 'Database_unresponsive']
 
 export interface SupportFormProps {
   setSentCategory: (value: string) => void
+  setSelectedProject: (value: string) => void
 }
 
-const SupportForm = ({ setSentCategory }: SupportFormProps) => {
+const SupportForm = ({ setSentCategory, setSelectedProject }: SupportFormProps) => {
```

Comment:
Is this even something you changed? Seems kinda unrelated and might just wrongfully show up here?

Reply:
No, that's not my changes.